src/Microsoft.Diagnostics.Monitoring.WebApi/Models/TraceProfile.cs

```diff
@@ -11,6 +11,7 @@ public enum TraceProfile
         Cpu = 0x1,
         Http = 0x2,
         Logs = 0x4,
-        Metrics = 0x8
+        Metrics = 0x8,
+        Gc = 0x10,
```

Comment:
```suggestion
        GcCollect = 0x10,
```
Let's match the name with the same profile name in the dotnet-trace tool: https://github.com/dotnet/diagnostics/blob/fe7763cdbd0ccbec8304887130bf3514a7f4f087/src/Tools/dotnet-trace/CommandLine/Commands/ListProfilesCommandHandler.cs#L65

There is a longer-term, there is a desire to make these exactly the same (https://github.com/dotnet/dotnet-monitor/issues/4606), but that should be a separate effort.